R/add_annotation.R

```diff
@@ -54,50 +54,54 @@ add_CellLine_annotation <- function(
   cellline_name <- gDRutils::get_env_identifiers("cellline_name")
   add_clid <- gDRutils::get_header("add_clid")
   
-  CLs_info <- data.table::fread(
-    system.file("annotation_data", fname, package = annotationPackage), header = TRUE
-  )
-  CLs_info <- CLs_info[, c(DB_cellid_header, DB_cell_annotate), with = FALSE]
-  
-  if (nrow(CLs_info) == 0) return(df_metadata)
-  
-  validatedCLs <- unique(df_metadata[[cellline]]) %in% CLs_info[[DB_cellid_header]]
-  missingTblCellLines <- NULL
-  if (!is.null(fill) && !all(validatedCLs)) {
-    unValidatedCellLine <- unique(df_metadata[[cellline]])[!validatedCLs]
-    missingTblCellLines <- data.table::data.table(
-      parental_identifier = unValidatedCellLine,
-      cell_line_name = unValidatedCellLine,
-      cell_line_identifier = unValidatedCellLine,
-      doubling_time = NA,
-      primary_tissue = fill,
-      subtype = fill
+  if (all(c(cellline, cellline_name, add_clid) %in% names(df_metadata))) {
+    return(df_metadata)
```

Comment:
as `df_metdata` is data.table, maybe it could be named `dt_metadata`?

Reply:
done in [a6a77cb](https://github.com/gdrplatform/gDRcore/pull/110/commits/a6a77cb246dca00d801f5857dedc604aa1146923)